lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,137 @@
+2.  [Start the Lagom sample application](#start-the-lagom-sample-application)
+3.  [Generate some traffic on the Lagom service](#generate-some-traffic-on-the-lagom-service)
+4.  [Analyse the traffic using IBM Project EventStore Notebooks](#analyse-the-traffic-using-ibm-project-eventstore-notebooks)
+
+## Download and set up the Lagom service
+
+(Make sure you completed all the steps on the Pre-Requisites section and IBM Project EventStore is installed and running)
+
+Follow these steps to get a local copy of this project and configure it to connect it to the IBM Project EventStore running on your machine
+
+1.  Open a command line shell and clone this repository:
+    ```
+    git clone https://github.com/lagom/ibm-integration-examples.git
+    ```
+2.  Change into the root directory for this example:
+    ```
+    cd lagom-eventstore-example
+    ```
+3.  To supply the configuration, do one of the following:
+    1. Open the `lagom-eventstore-impl/src/main/resources/ibm-event-store.conf` file in a text editor and fill in the empty value of the `endpoints` setting.
+    2. On the same file, provide the value of the `db.name` setting. The `db.name` is not hardcoded because IBM Project EventStore can only hold one database at the same time. If you already have a database and its name is different from the name you want Lagom to use you will have to remove the database from the IBM Project EventStore manually. This example application may rebuild the EventStore from scratch on every reboot so you can test from a clean slate: use the `clear-schema` config to rebuild the database from scratch on every run.
+    3. If you prefer not to enter credentials into the file, you can also set them as environment variables named `IBM_EVENTSTORE_ENDPOINTS`,  and `IBM_EVENTSTORE_DATABASE_NAME`.
+4. Download and install the IBM Project Event Store client library.
+    1. Make sure you are on the project folder
+    2. Download and unzip the client library
+    ```
+wget https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/download/1.1.1/ibm-eventstore-developer-client.1.1.1.zip
+unzip ibm-eventstore-developer-client.1.1.1.zip
+    ```
+    3. Install the library on your local repository:
+    ```
+mvn install:install-file -Dfile=ibm-eventstore-developer-client.1.1.1/ibm-event.jar -Dversion=1.1.1 -DgroupId=com.ibm.eventstore -DartifactId=ibm-event -Dpackaging=jar
+    ```
+
+
+## Start the Lagom sample application
+
+In the command line shell where you downloaded the Lagom service, from the `lagom-eventstore-example` directory, start the Lagom development environment by running:
+
+```
+mvn lagom:runAll
+```
+
+You should see some console output, including these lines:
+
+```
+...
+[INFO] Service gateway is running at http://localhost:9000
+...
+[INFO] (Service started, press enter to stop and go back to the console...)
+```
+
+These messages indicate that the service has started correctly.
```

Comment:
The amount of output from the service is kind of overwhelming. Can we tune the log configuration?